zeppelin-server/pom.xml

```diff
@@ -43,19 +43,19 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>2.11.8</version>
```

Comment:
@Leemoonsoo - Sorry for the trouble. I found that my .local.bash had "export CLASSPATH" which caused the interpreter.sh to pick up zeppelin server classpath. I reverted the scalding version change to zeppelin-server/pom.xml. Thanks!